actiontext/app/assets/javascripts/trix.js

```diff
@@ -1,6 +1,6 @@
 /*
 Trix 2.0.0
-Copyright © 2022 Basecamp, LLC
+Copyright © Basecamp, LLC
```

Comment:
This is a vendored file, we probably shouldn't modify it

Reply:
@skipkayhil Good catch, forgot this was vendored. Rebased!